abesid-to-etab/index.js

```diff
@@ -40,10 +41,11 @@ module.exports = function () {
   const sourceField = req.header('abesid-to-etab-source-field') || 'abes-id';
 
   const enrichedField = req.header('abesid-to-etab-enriched-field') || 'institutionName';
+  const filenameField = req.header('abesid-to-etab-filename-field') || 'Etablissements.csv';
```

Comment:
The header is not about a `field` of the CSV, I guess we can juste name it `filename`

```suggestion
  const filenameField = req.header('abesid-to-etab-filename') || 'Etablissements.csv';
```

Reply:
My bad